Formula/bookloupe.rb

```diff
@@ -0,0 +1,62 @@
+class Bookloupe < Formula
+  desc "Lists common formatting errors in a Project Gutenberg candidate file"
+  homepage "http://www.juiblex.co.uk/pgdp/bookloupe/"
+  url "http://www.juiblex.co.uk/pgdp/bookloupe/bookloupe-2.0.tar.gz"
+  sha256 "15b1f5a0fa01e7c0a0752c282f8a354d3dc9edbefc677e6e42044771d5abe3c9"
+
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    ln_s("#{bin}/bookloupe", "./bookloupe")
+
+    test_list = [
+      "abbreviation",
+      "before-comma",
+      "before-period",
+      "brackets",
+      "control-characters",
+      "dashes",
+      "digits",
+      "double-punctuation",
+      "ellipsis",
+      "embedded-cr",
+      "embedded-lf",
+      "example",
+      "extra-period",
+      "genatives",
+      "grave-quotes",
+      "he-be",
+      "hebe-jeebies",
+      "html-symbol",
+      "html-tag",
+      "long-line",
+      "mail-from",
+      "mismatched-quotes",
+      "missing-space",
+      "non-ascii",
+      "periods",
+      "scannos",
+      "short-line",
+      "single-quotes",
+      "spaced-doublequote",
+      "spaced-punctuation",
+      "unmarked-paragraph",
+      "unusual-characters",
+      "user-defined-typo",
+      "windows-1252",
+    ]
```

Comment:
No, you can't unfortunately. One test fails on Mac (passes on Linux). I omitted that test here since homebrew only is used on Macs, and opened a bug with the author about it. I'll switch to `%w[]`.